Kitodo/src/main/java/de/sub/goobi/forms/DocketForm.java

```diff
@@ -144,4 +145,17 @@ public int getDocketId() {
     public void setDocketId(int id) {
         this.docketId = id;
     }
+
+    // TODO:
+    // replace calls to this function with "/pages/projects" once we have completely
+    // switched to the new frontend pages
+    private String redirectToList(String urlSuffix) {
+        String referrer = FacesContext.getCurrentInstance().getExternalContext().getRequestHeaderMap().get("referer");
+        String callerViewId = referrer.substring(referrer.lastIndexOf("/") + 1);
+        if (!callerViewId.isEmpty() && callerViewId.contains("editDocket.jsf")) {
+            return "/pages/projects" + urlSuffix;
```

Comment:
Why are you returning to `/pages/projects`? Maybe a wrong copy and past?

Reply:
No, this path is correct, since "dockets" are now located in an additional tab of the "projects" page instead of having their own, dedicated page.